corehq/toggles.py

```diff
@@ -1727,6 +1727,14 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_USER],
 )
 
+ICDS_NIC_INDICATOR_API = StaticToggle(
```

Comment:
I think better would be like `ICDS_GOVERNANCE_DASHABOARD`. as feature flag for NIC api is already there.

Reply:
done